host_core/lib/host_core/host.ex

```diff
@@ -52,6 +51,7 @@ defmodule HostCore.Host do
 
   @impl true
   def handle_continue(:query_lattice_cache, state) do
+    Process.sleep(300)
```

Comment:
This will likely guarantee that a host can request itself for the lattice cache, feels wrong to me. I think you can remove this `Process.sleep` without changing the fact that we are not guaranteeing proper cache retrieval. Not sure how far jetstream is away from us now, but I don't like that this leaves proper cache retrieval up to getting lucky on the queue subscribe

Reply:
So the core part of the problem here is that there's a race condition, without this sleep it's entirely possible to attempt to request the cache contents before the connection supervisor(s) have prepared the NATS connection(s). Using the connection supervisor versus a regular connection apparently introduces some delay in when the connection starts.

Reply:
Without this, I can get the tests to fail locally on my machine about 30% of the time just by attempting to query the cache before the NATS connections are available.

Reply:
we should, in theory, have a version of `Gnat` that talks to JetStream in a week or so.

Reply:
Ah so the issue is that we can't actually _request_ the claims because the nats connection isn't ready yet, but if we wait for the nats connection then we might actually have already subscribed to the linkdef.get. 

Stupid computers.